src/test/java/org/onosproject/monitoring/server/Client.java

```diff
@@ -29,16 +28,23 @@ public static void main(String args[]){
         } catch (UnknownHostException e) {
             e.printStackTrace();
         }
-        buf= "hello".getBytes();
+
+
+
+        buf = PacketTestUtils.readFromPcapFile("/home/iver/prometheus_int_exporter/src/test/resources/int_report.pcap");
```

Comment:
Buffer bytes should start from the UDP portion, since we're sending the packet through the socket the OS also adds its own Ethernet and IP headers.